modules/nf-core/tiledbvcf/create/main.nf

```diff
@@ -0,0 +1,33 @@
+
+process TILEDBVCF_CREATE {
+    tag "$uri"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "tiledb::tiledbvcf-py=0.21.0" : null)
+    container "${ 'tiledb/tiledbvcf-cli:0.21.0' }"
+    containerOptions '--entrypoint ""'
```

Comment:
What's this do?

Reply:
We add a directive to specify container executions options. With the absence of the `--entrypoint ""` the call to the tiledbvcf command does not recognize subcommands.

Reply:
This was figured out by @leipzig :)